src/test/java/com/hocs/test/glue/CreateCaseStepDefs.java

```diff
@@ -86,8 +89,40 @@ public void aCaseIsCreated() {
         page.sleep(500);
     }
 
-    @When("^I create a case with <Topic>$")
-    public void aCaseWithSpecificTopicIsCreated() {
+    @When("^I create a \"([^\"]*)\" case with \"([^\"]*)\"$")
+    public void aCaseWithSpecificTopicIsCreated(String caseType, String topic) {
+        switch (caseType.toUpperCase()) {
+            case "DCU MIN":
+                createCase.createDCUMinSingleCase();
+                setSessionVariable("caseType").to(caseType);
+                dataInput.dataInputFullFlow();
+                topics.fromMarkupStartSelectATopic(topic);
+                break;
+            default:
+                System.out.println(caseType
```

Comment:
```suggestion
                pendingStep(pageName + " is not defined within " + getMethodName());
```

N.B. this will require a static import for `pendingStep`.